package/gluon-wireless-encryption/Makefile

```diff
@@ -0,0 +1,33 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-wireless-encryption
+PKG_VERSION:=1
```

Comment:
Drop `PKG_VERSION`, `PKG_RELEASE` is sufficient.

Reply:
`PKG_VERSION` is still here, did you miss this?

Reply:
Faulty push, fixed.